tests/providers/provider.js

```diff
@@ -727,6 +727,28 @@ test('setting skipIfEmpty to false runs only the first rule', async () => {
   expect(errors.at(0).text()).toBe('The {field} field must be a valid email');
 });
 
+test('setting detectInput to false disables the v-model autodetection', async () => {
+  const wrapper = mount(
+    {
+      data: () => ({
+        value: ''
+      }),
+      template: `
+        <ValidationProvider :detectInput="false" rules="required" v-slot="{ errors }">
+          <input v-model="value" type="text">
+          <span id="error">{{ errors[0] }}</span>
+        </ValidationProvider>
+      `
+    },
+    { localVue: Vue, sync: false }
+  );
+
+  await flushPromises();
+
+  const error = wrapper.find('#error');
+  expect(error.text()).toBeFalsy();
```

Comment:
Great, one last step. Your test currently doesn't verify the new behavior because the input wasn't updated/changed.

So before making that assertion, make sure to change the input value, first start with a valid value like `123` then add the following before the assertion:

```js
const input = wrapper.find('#input');
input.setValue('');
await flushPromises();
expect(error.text()).toBeFalsy();
```

Reply:
Done!